README.md

```diff
@@ -852,21 +852,42 @@ This output writes data to a webserver using diffent encoding methods (Json or J
 ### Event Hub
 *Nuget Package*: [**Microsoft.Diagnostics.EventFlow.Outputs.EventHub**](https://www.nuget.org/packages/Microsoft.Diagnostics.EventFlow.Outputs.EventHub/)
 
-This output writes data to the [Azure Event Hub](https://azure.microsoft.com/en-us/documentation/articles/event-hubs-overview/). Here is an example showing all possible settings:
+> **BREAKING CHANGE**: Starting from version `2.0.0` this package targets _only_ `netstandard2.0`.
```

Comment:
There is a "Platform Support" section at the end of the document; I think the information about it should go there. Namely, that version 1.x of the output package support 4.5.1, 4.6.1 and .NET Core frameworks, but version 2.x supports only 4.6.1 and .NET Core/.NET 5 frameworks

Reply:
Updated "Plarform Support".